src/rules/heading-blank-lines.ts

```diff
@@ -93,11 +93,12 @@ export default class HeadingBlankLines extends RuleBuilder<HeadingBlankLinesOpti
       }),
       new ExampleBuilder({
         // accounts for https://github.com/platers/obsidian-linter/issues/219
-        description: 'Empty line before header and after Yaml is removed with `Empty Line Between Yaml and Header=true`',
+        description: 'Empty line before header and after Yaml is removed with `Empty Line Between Yaml and Header=false`',
         before: dedent`
           ---
           key: value
           ---
+          
```

Comment:
In order to make this addition you should be setting the blank line like so (I see it was not there before, but that was standard decided upon to help make it clear what whitespace is intended since dedent and eslint rules around whitespace are in use):
```suggestion
          ${''}
```